lib/content/index.js

```diff
@@ -62,6 +62,7 @@ const rootModule = {
 const workspaceRepo = {
   add: {
     ...releasePlease(true),
+    '.github/matchers/tap.json': 'tap.json',
```

Comment:
It's safe to put this file in both root and workspace settings since the content is always the same, so it will only get written once. We need to do this because it's possible for only workspaces to use template-oss in which case we would be missing this file.